src/test/java/pl/edu/icm/pl/mxrdr/extension/workflow/handler/MxrdrNotificationSenderTest.java

```diff
@@ -0,0 +1,160 @@
+package pl.edu.icm.pl.mxrdr.extension.workflow.handler;
+
+import edu.harvard.iq.dataverse.engine.command.DataverseRequest;
+import edu.harvard.iq.dataverse.mail.EmailContent;
+import edu.harvard.iq.dataverse.mail.MailService;
+import edu.harvard.iq.dataverse.persistence.dataset.Dataset;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetField;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetFieldType;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetVersion;
+import edu.harvard.iq.dataverse.persistence.dataset.FieldType;
+import edu.harvard.iq.dataverse.persistence.group.IpAddress;
+import edu.harvard.iq.dataverse.persistence.user.ApiToken;
+import edu.harvard.iq.dataverse.persistence.user.AuthenticatedUser;
+import edu.harvard.iq.dataverse.persistence.user.UserNotification;
+import edu.harvard.iq.dataverse.persistence.user.UserNotificationDao;
+import edu.harvard.iq.dataverse.persistence.workflow.Workflow;
+import edu.harvard.iq.dataverse.persistence.workflow.WorkflowExecution;
+import edu.harvard.iq.dataverse.workflow.WorkflowContext;
+import edu.harvard.iq.dataverse.workflow.WorkflowExecutionContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.edu.icm.pl.mxrdr.extension.notification.MxrdrNotificationType;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static edu.harvard.iq.dataverse.workflow.WorkflowContext.TriggerType.PostPublishDataset;
+import static java.util.Collections.emptyMap;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+@ExtendWith(MockitoExtension.class)
+public class MxrdrNotificationSenderTest {
+
+    private MxrdrNotificationSender mxrdrNotificationSender;
+
+    private UserNotificationDao userNotificationDao;
+
+    private MailService mailService;
+
+    private AuthenticatedUser user;
+    private final Instant timer = Instant.ofEpochSecond(1592570489);
```

Comment:
This `1592570489` is a magic number. Pleas use sth. like `Instant.parse("2020-06-01T09:10:20.00Z")` instead to make it readable. Or better yet `Clock clock = Clock.fixed(Instant.parse("2020-06-01T09:10:20.00Z"), UTC);`